source/operations/checklists/software.rst

```diff
@@ -8,4 +8,62 @@ Software Checklist
    :local:
    :depth: 2
 
-Stub - detail software checklist items
+Use the following checklist when planning the software configuration for a production, distributed MinIO deployment.
+
+Pre-MinIO Install Requirements
+------------------------------
+
+.. list-table::
+   :widths: auto
+   :width: 100%
+
+   * - :octicon:`circle`
+     - Servers running a Linux operating system, such as a supported version of Red Hat Enterprise Linux (RHEL) or Ubuntu LTS
+
+   * - :octicon:`circle` 
+     - System administrator access to the remote servers
+
+   * - :octicon:`circle`
+     - A distributed deployment solution, such as `Podman <https://podman.io>`__ for containers or `kubectl <https://kubernetes.io/docs/tasks/tools/#kubectl>`__ for Kubernetes
+
+   * - :octicon:`circle`
+     - Load balancer to handle routing of requests (for example, `NGINX <https://www.nginx.com/>`__)
+
+   * - :octicon:`circle`
+     - :ref:`Prometheus / Grafana <minio-metrics-collect-using-prometheus>` setup
+
+   * - :octicon:`circle`
+     - :ref:`Security Token Service (STS) <minio-external-identity-management-ad-ldap>` defined
```

Comment:
STS also works for OIDC. And you can't do this w/o MinIO, so I'd drop this or move it up to the security section as a checklist item for external IDPs - specifically that applications can use the STS API (either via REST or SDK) to authenticate to MinIO.

Reply:
Added a 3rd Party IDp Tasks section and moved to there.